pymc3/step_methods/hmc/nuts.py

```diff
@@ -115,14 +115,11 @@ def __init__(self, vars=None, Emax=1000, target_accept=0.8,
         adapt_step_size : bool, default=True
             Whether step size adaptation should be enabled. If this is
             disabled, `k`, `t0`, `gamma` and `target_accept` are ignored.
-        adapt_mass_matrix : bool, default=True
-            Whether to adapt the mass matrix during tuning if the
-            potential supports tuning.
         max_treedepth : int, default=10
             The maximum tree depth. Trajectories are stoped when this
             depth is reached.
         early_max_treedepth : int, default=8
-            The maximum tree depth during tuning.
+            The maximum tree depth during the first 200 tuning samples.
```

Comment:
I really hope people don't use less than 200 tuning steps :-)
I think "during the first 200 tuning samples" would imply that, no?